lib/services.credentials.js

```diff
@@ -216,10 +291,7 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
       filteredCredential: ['record', 'issuer', 'holder',
         function(callback, results) {
         var credential = results.record.credential;
-        // FIXME
-        logger.warning('FIXME: skipping get credential perm checks!');
-        //if(results.issuer || results.holder) {
-        if(true) {
+        if(results.issuer || results.holder) {
```

Comment:
This should be done through the permissions API. A permission check can be performed by mapping the resource to either the issuer or the holder:

https://github.com/digitalbazaar/bedrock-permission/blob/master/lib/permission.js#L122-L123